src/box/watcher.c

```diff
@@ -364,15 +369,27 @@ watchable_create(struct watchable *watchable)
 	watchable->node_by_key = mh_strnptr_new();
 	rlist_create(&watchable->pending_watchers);
 	watchable->worker = NULL;
+	watchable->is_shutdown = false;
 }
 
+/**
+ * Shutdown watchable. After shutdown it is still can be used but
```

Comment:
Nit: `is` is redundant. Same comment for `box_watcher_shutdown`.

Reply:
Fixed.

```diff
--- a/src/box/watcher.c
+++ b/src/box/watcher.c
@@ -371,7 +371,7 @@ watchable_create(struct watchable *watchable)
 }

 /**
- * Shutdown watchable. After shutdown it is still can be used but
+ * Shutdown watchable. After shutdown it still can be used but
  * notifications are stopped.
  */
 static void
diff --git a/src/box/watcher.h b/src/box/watcher.h
index 766948b3a..4aaf2275b 100644
--- a/src/box/watcher.h
+++ b/src/box/watcher.h
@@ -288,7 +288,7 @@ void
 box_watcher_init(void);

 /**
- * Shutdown watcher. After shutdown API is still can be used but notifications
+ * Shutdown watcher. After shutdown API still can be used but notifications
  * are stopped.
  */
 void
```